assets/js/modules/analytics/dashboard/dashboard-widget-all-traffic.js

```diff
@@ -112,11 +113,11 @@ class AnalyticsAllTraffic extends Component {
 
 				</div>
 				{ this.getErrorDataComponent() }
-				<div className={ `
+				<div className={ classnames( `
 					mdc-layout-grid__cell
-					mdc-layout-grid__cell--span-12
-					${ wrapperClass }
-				` }>
+					mdc-layout-grid__cell--span-12`,
+				wrapperClass
```

Comment:
Oh, I see, these strings should each be their own string/line, not a large template string.